Netflix Clone/Services/APIClient.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  APIClient.swift
+//  Netflix Clone
+//
+//  Created by Polina Poznyak on 6.07.23.
+//
+
+import Foundation
+
+struct APIClient {
+    static let shared = APIClient()
+    let session = URLSession(configuration: .default)
```

Comment:
```suggestion
    private let session = URLSession(configuration: .default)
```
If you have APIClient, it makes sense to hide the session